.github/workflows/main.yaml

```diff
@@ -121,7 +130,7 @@ jobs:
       - name: Release
         id: release
         #TODO: use a hash target when ready for merge
-        uses: SonarSource/gh-action_release/main@feat/BUILD-4602-Burgr-independant-releasability
+        uses: SonarSource/gh-action_release/main@feat/mm/build-4609-check-releasability
```

Comment:
make sense, let's updated it on the target branch after merge otherwise it will fail the checks in this PR